### PR TITLE
add QuestionController, DTO and mapper

### PR DESCRIPTION
**Pull request by volodymyr-chornyi**

### CODE REVIEW

File: src/main/java/com/softserve/academy/event/controller/QuestionController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@RequestMapping("question")
+public class QuestionController {
+
+    @Autowired
+    private QuestionService questionService;
+
+    @Autowired
+    private QuestionMapper questionMapper;
+
+    @RequestMapping(method = RequestMethod.GET)
```
Comment (clevercattv) [inline src/main/java/com/softserve/academy/event/controller/QuestionController.java]: @RequestMapping(method = RequestMethod.GET) == @GetMapping

File: src/main/java/com/softserve/academy/event/repository/impl/QuestionRepositoryImpl.java [hunk 2]
```diff
@@ -3,7 +3,19 @@
 import com.softserve.academy.event.entity.SurveyQuestion;
 import com.softserve.academy.event.repository.QuestionRepository;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
 
 @Repository
-public class QuestionRepositoryImpl extends BasicRepositoryImpl<SurveyQuestion, Long> implements QuestionRepository {
+@Transactional
```
Comment (clevercattv) [inline src/main/java/com/softserve/academy/event/repository/impl/QuestionRepositoryImpl.java]: Transactional set's to service, not repository

File: src/main/java/com/softserve/academy/event/controller/QuestionController.java (hunk 1 above)
Comment (volodymyr-chornyi) [reply src/main/java/com/softserve/academy/event/controller/QuestionController.java]: fixed

File: src/main/java/com/softserve/academy/event/repository/impl/QuestionRepositoryImpl.java (hunk 2 above)
Comment (volodymyr-chornyi) [reply src/main/java/com/softserve/academy/event/repository/impl/QuestionRepositoryImpl.java]: fixed

**Review by clevercattv — APPROVED** [review]

**Review by Qwe1999 — APPROVED** [review]

**Pull request merged**